src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -141,40 +140,34 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
 
   return (
     <>
-      <ScrollView
-        contentContainerStyle={{ alignItems: "center" }}
-        scrollIndicatorInsets={{ right: 1 }}
-        keyboardShouldPersistTaps="handled"
-      >
+      <Credits style={{ bottom: size(3) }} />
```

Comment:
Ooooh can you explain why this becomes on top of the scrolling view?

Reply:
Oh because of the z-index, previously when the keyboard opens, given that the viewheight decreases, the absoluted credits will shift up and block some content